test/unit/ngsiv2/plugins/alias-plugin_test.js

```diff
@@ -144,6 +180,239 @@ describe('Attribute alias plugin', function() {
                 .reply(204);
         });
 
+        it('should rename the attributes as expected by the mappings', function(done) {
+            iotAgentLib.update('light1', 'Light', '', values, function(error) {
+                should.not.exist(error);
+                contextBrokerMock.done();
+                done();
+            });
+        });
+    });
+    describe('When an update comes for attributes with aliases and integer type', function() {
```

Comment:
Given the evolution of this PR I think the "with aliases" part should be removed in this and the rest of describes.

Reply:
...or maybe not :)

I mean, I have just realized that you are using a couple of test cases for atributes. One with alias, the other witout. Thus, every fine.

NTC

Reply:
However, the file is named alias-test.js...

Maybe is a good idea to leave alias-test.js as it was and put the new test cases in a separated autocast-test.js file (or similar name). What do you think?

Reply:
I would maintain the new content of alias_plugin_test.js because it allows checking the behavior of the casting functionality when attributes are reported using the alias.
In addition, I will follow your proposal and I will add a new autocast-test.js file

Reply:
Fixed in d89d8672aeb0937e099e769d4ad405e4cc59183a